app/components/modals/session-notify-modal.js

```diff
@@ -63,8 +63,8 @@ export default class SessionNotifyModal extends ModalBase {
       payload.message = newMessage;
     }
 
-    if (this.bcc) {
-      payload.bcc = this.bcc;
+    if (this.bccString) {
+      payload.bcc = this.bccString.split(', ');
```

Comment:
I think first split from `,` and trim the space from result would be a better logic? because may be someone enters email like `hope@gmail.com,hope1@gmail.com , hope3@gmail.com`

Reply:
payload.bcc = this.bccString.replace(/s/g, '').split(', ');